webpack/JobInvocationDetail/JobInvocationDetail.scss

```diff
@@ -0,0 +1,42 @@
+$chart_size: 105px;
```

Comment:
This also should be in `.job-invocation-detail-page-section` or have unique name